src/odemis/driver/xt_client.py

```diff
@@ -376,6 +377,24 @@ def move_stage(self, position: Dict[str, float], rel: bool = False) -> None:
             self.server._pyroClaimOwnership()
             self.server.move_stage(position, rel)
 
+    def set_raw_coordinate_system(self, raw_coordinates:bool = True):
+        """
+        Read raw z coordinate or linked z coordinate when requesting stage coordinates
+        :param raw_coordinates: (bool) If True, request raw z stage coordinates, otherwise request linked Z coordiantes
+        """
+        try:
+            with self._proxy_access:
+                self.server._pyroClaimOwnership()
+                self.server.set_raw_coordinate_system(raw_coordinates)
+        except AttributeError:
+            if raw_coordinates:
+                raise NotImplementedError("Ensure the version of Delmic XT Adapter is >= 1.12.0")
+            # If non-raw coordinates are requested, proceed with the current coordinate system
```

Comment:
```suggestion
                raise NotImplementedError("Ensure the version of Delmic XT Adapter is >= 1.12.0")
            # Do not raise an error if non-raw coordinates are requested, because non-raw is the default in old versions of the xtadapter
```